Userland/Libraries/LibWeb/CSS/Enums.json

```diff
@@ -144,6 +144,10 @@
     "zoom-in",
     "zoom-out"
   ],
+  "direction": [
+    "ltr",
+    "rtl"
+  ],
```

Comment:
Can you please also modify the `direction` entry in `Properties.json` to use this?

```diff
diff --git a/Userland/Libraries/LibWeb/CSS/Properties.json b/Userland/Libraries/LibWeb/CSS/Properties.json
index 1a89a8ecb10..a3b2a70f465 100644
--- a/Userland/Libraries/LibWeb/CSS/Properties.json
+++ b/Userland/Libraries/LibWeb/CSS/Properties.json
@@ -1065,9 +1065,8 @@
     "animation-type": "none",
     "inherited": true,
     "initial": "ltr",
-    "valid-identifiers": [
-      "ltr",
-      "rtl"
+    "valid-types": [
+      "direction"
     ]
   },
   "display": {
```

Reply:
Sorry, I'm familiar with Git, not so much with GitHub. I wanted to convert it to a draft because I released I’d made a mistake, but pressed close on accident. Anyway, thanks for reviewing, and I’ve made that change.

Reply:
Ah gotcha. Easily done. :sweat_smile: